development-docs/systemtests/io.strimzi.systemtest.log.LoggingChangeST.md

```diff
@@ -248,6 +248,28 @@
 * [logging](labels/logging.md)
 
 
+## testJsonTemplateLayoutFormatLogging
```

Comment:
@im-konge @see-quick Do we really want to have tests covering both JSON layouts? I think we should pick one and focus on it ... and remove the testing for the other. As it seems we decided to focus on the JSON Layout module, we should focus on that?

Reply:
That is strange. I would expect the test to be different for 3.9 and 4.0 given Log4j1 versus 2.

Reply:
Ah, ok.